src/components/ebay-pagination/test/test.browser.js

```diff
@@ -0,0 +1,108 @@
+const sinon = require('sinon');
+const expect = require('chai').expect;
+const testUtils = require('../../../common/test-utils/browser');
+const mock = require('../mock');
+const renderer = require('../');
+
+describe('given the menu is in the default state with normal links', () => {
+    let widget;
+    let root;
+    let previousButton;
+    let pageItem;
+    let nextButton;
+
+    beforeEach(() => {
+        widget = renderer.renderSync(mock.basicLinks).appendTo(document.body).getWidget();
+        root = document.querySelector('nav.pagination');
+        previousButton = root.querySelector('.pagination__previous');
+        nextButton = root.querySelector('.pagination__next');
+        pageItem = root.querySelector('.pagination__item');
+    });
+    afterEach(() => widget.destroy());
+
+    describe('when the previous button is clicked', () => {
+        let spy;
+        beforeEach(() => {
+            spy = sinon.spy();
+            widget.on('pagination-previous', spy);
+            testUtils.triggerEvent(previousButton, 'click');
+        });
+
+        test('then it emits the marko event called pagination-previous', () => {
+            expect(spy.calledOnce).to.equal(true);
+            expect(Object.keys(spy.args[0][0])).to.deep.equal(['el']);
+        });
+    });
+
+    describe('when the next button is clicked', () => {
+        let spy;
+        beforeEach(() => {
+            spy = sinon.spy();
+            widget.on('pagination-next', spy);
+            testUtils.triggerEvent(nextButton, 'click');
+        });
+
+        test('then it emits the marko event called pagination-next', () => {
+            expect(spy.calledOnce).to.equal(true);
+            expect(Object.keys(spy.args[0][0])).to.deep.equal(['el']);
+        });
+    });
+
+
+    describe('when the page number is clicked', () => {
+        let spy;
+        beforeEach(() => {
+            spy = sinon.spy();
+            widget.on('pagination-select', spy);
+            testUtils.triggerEvent(pageItem, 'click');
+        });
+
+        test('then it emits the marko event called pagination-select', () => {
+            expect(spy.calledOnce).to.equal(true);
+            expect(Object.keys(spy.args[0][0])).to.deep.equal(['el', 'value']);
```

Comment:
Can you test the actual values rather than just the keys? I.e. `expect(eventData.value).to.equal(x)`

Reply:
Okay will update these